ihatemoney/web.py

```diff
@@ -895,12 +896,23 @@ def strip_ip_addresses():
 @main.route("/<project_id>/statistics")
 def statistics():
     """Compute what each member has paid and spent and display it"""
-    today = datetime.now()
+    if g.project.has_bills():
+        first_date = g.project.get_least_recent_bill().date
+        last_date = g.project.get_most_recent_bill().date
+    else:
+        first_date = datetime.now() - relativedelta(months=12)
+        last_date = datetime.now()
```

Comment:
I don't quite understand why we need to have last and first date when we don't have bills yet?

Reply:
This is just to mirror the current behaviour (last one year), but yes, we can simply display nothing in this case.

Reply:
Changed to display nothing instead.